common/src/main/scala/com/gu/media/pluto/PlutoProjectDataStore.scala

```diff
@@ -1,36 +1,33 @@
 package com.gu.media.pluto
 
-import com.amazonaws.services.dynamodbv2.model.DeleteItemResult
 import com.gu.media.aws.DynamoAccess
 import com.gu.media.logging.Logging
 import com.gu.media.pluto.PlutoItem.numericIdsOnlyFilter
 import org.joda.time.{DateTime, DateTimeZone}
 import org.scanamo.DynamoFormat._
+import org.scanamo.generic.auto._
 import org.scanamo.syntax._
-import org.scanamo.{DynamoFormat, Scanamo, Table}
-import org.scanamo.auto._
+import org.scanamo.{DynamoFormat, DynamoReadError, Scanamo, Table}
 
 case class PlutoProjectDataStoreException(err: String) extends Exception(err)
 
 class PlutoProjectDataStore(aws: DynamoAccess, plutoCommissionDataStore: PlutoCommissionDataStore) extends Logging {
-  implicit val dateTimeFormat = DynamoFormat.coercedXmap[DateTime, String, IllegalArgumentException](
-    DateTime.parse(_).withZone(DateTimeZone.UTC)
-  )(_.toString)
+  implicit val dateTimeFormat: DynamoFormat[DateTime] = DynamoFormat.coercedXmap[DateTime, String, IllegalArgumentException](
+    DateTime.parse(_).withZone(DateTimeZone.UTC),
+    _.toString
+  )
```

Comment:
This change prompted by https://github.com/scanamo/scanamo/pull/588 - apparently this gave better type inference, but I'm not sure how exactly...!